pkg/sink/kafka/options.go

```diff
@@ -151,6 +151,7 @@ type Options struct {
 	ReplicationFactor int16
 	Version           string
 	IsAssignedVersion bool
+	RequestVersion    int16
```

Comment:
Do we need to verify the value range of the RequestVersion?

Reply:
No, it only use in unit test

Reply:
Then, maybe we should use a const variable and set it in only in tests.

Reply:
it will be 0 as default, and the unit test needs to be 3.